datasets/allen-cell-imaging-collections.yaml

```diff
@@ -0,0 +1,20 @@
+Name: Allen Cell Imaging Collections
+Description: |
+  https://allencell.quiltdata.com/b/quilt-aics
+
+  This bucket contains multiple datasets (as Quilt packages) created by the Allen Institute for Cell Science. The imaging data in this bucket contains either of the following: 1) field of view images from glass plates, 2) cell membrane, DNA, and structure segmentations, 3) cell membrane, DNA and structure contours, 4) machine learning imaging predictions of the previously listed modalities. In addition, many of the datasets include CSVs that contain feature sets related to that data.
```

Comment:
Whole thing looks good. For the above string, can you break it into multiple lines so that it fits in ~80 character terminal?